cloud/pkg/csidriver/utils.go

```diff
@@ -139,10 +140,11 @@ func logGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, h
 
 // buildResource return a string as "beehive/pkg/core/model".Message.Router.Resource
 func buildResource(nodeID, namespace, resourceType, resourceID string) (string, error) {
-	if nodeID == "" || namespace == "" || resourceType == "" {
```

Comment:
The namespace here is refer to node namespace?

Reply:
In fact, i find that in the beehive side, no one uses namespace.